tests/integration/misbehaviour.go

```diff
@@ -385,27 +385,55 @@ func (s *CCVTestSuite) TestCheckMisbehaviour() {
 	altSigners[clientTMValset.Validators[0].Address.String()] = clientSigners[clientTMValset.Validators[0].Address.String()]
 	altSigners[clientTMValset.Validators[1].Address.String()] = clientSigners[clientTMValset.Validators[1].Address.String()]
 
+	// create an alternative validator set using less than 1/3 of the trusted validator set
+	altValset2 := tmtypes.NewValidatorSet(s.consumerChain.Vals.Validators[0:1])
```

Comment:
nit: should we move those after `clientHeader` because they're not used in `clientHeader`?